packages/dom-helper/lib/prop.js

```diff
@@ -59,5 +58,49 @@ function isSettable(element, attrName) {
     }
   }
 
+  // Properties can be effectively read-only two ways.
+  // If actually marked as writable = false, an exception is thrown if you attempt
+  // to assign. If it's simply missing a setter, it silently just doesn't
+  // assign anything. Both cases we will defer to setAttribute instead
+  var desc = getPropertyDescriptor(element, attrName);
+  if (!desc) { return true; }
+  if (desc.writable === false || !desc.hasOwnProperty('value') && typeof desc.set !== 'function') {
```

Comment:
@rwjblue and I are discussing this in Slack...but this `typeof desc.set !== 'function'` fails in Safari due to this outstanding bug: https://bugs.webkit.org/show_bug.cgi?id=49739. tl;dr Safari is the only browser that doesn't return full descriptors for host element properties.

Chrome:
![image](https://cloud.githubusercontent.com/assets/762949/8094984/6b40c5f0-0f7f-11e5-94ec-56ef3df12cc2.png)

Safari:
![image](https://cloud.githubusercontent.com/assets/762949/8094990/71927cfa-0f7f-11e5-924f-7d4b3af148ee.png)